hilti/runtime/include/types/interval.h

```diff
@@ -31,7 +31,7 @@ class Interval {
      *
      * @param nsecs interval in nanoseconds.
      */
-    explicit Interval(hilti::rt::integer::safe<int64_t> nsecs, NanosecondTag /*unused*/) : _nsecs(nsecs) {}
+    explicit Interval(const hilti::rt::integer::safe<int64_t>& nsecs, NanosecondTag /*unused*/) : _nsecs(nsecs) {}
```

Comment:
All the changes in this patch going from passing a safe int by value to passing by ref seem completely unnecessary to me since a `safe<int64_t>` should have exactly the same size as a `int64_t`. My clang-tidy version also seems to be happy with the existing code. What warning did this trigger and what version of clang-tidy were you running?

Reply:
I saw this on CI originally, see https://cirrus-ci.com/task/4783218440798208. Locally I'm using clang-tidy 15.0.7 currently.